### PR TITLE
Stop changesets from creating github releases

### PR DESCRIPTION
**Pull request by trianglesphere**
**Description**

Stops changesets from creating github releases for canary releases.

### CODE REVIEW

**Review by tynes — APPROVED** [review]
Confirming this is the correct config https://github.com/changesets/action#inputs

**Review by roninjin10 — APPROVED** [review]

**Review by roninjin10 — CHANGES_REQUESTED** [review]
I think you need to add this to line 70 to do what you want this to do

File: .github/workflows/publish-canary.yml [hunk 1]
```diff
@@ -75,6 +75,7 @@ jobs:
         uses: changesets/action@v1
         id: changesets
         with:
+          createGithubReleases: false
           publish: yarn changeset publish --tag canary
```
Comment (tynes) [inline .github/workflows/publish-canary.yml:79]: oh good call @roninjin10 the canary snapshot one

Comment (roninjin10) [reply .github/workflows/publish-canary.yml:79]: Yea not sure I think line 70 might need it I just approved to @trianglesphere figure it out

Comment (trianglesphere) [reply .github/workflows/publish-canary.yml:79]: Added to the full release. TY for the find.

**Review by clabby — APPROVED** [review]
LGTM

**Review by mslipper — APPROVED** [review]

**Pull request merged**